TombEngine/Specific/IO/lz4.cpp

```diff
@@ -0,0 +1,2924 @@
+/*
+   LZ4 - Fast LZ compression algorithm
```

Comment:
I remind you that external libraries should be put into `Libs` folder, not in the `Specific` folder.
Also, please consider using dll file instead of statically linking the library, if it is possible.